### PR TITLE
STAR-510 add dashboard cards

### PR DESCRIPTION
**Pull request by oleynichenko**
Screenshot (37)

### CODE REVIEW

File: src/components/common/custom-button.js [hunk 1]
```diff
@@ -4,13 +4,15 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
...
   return {
     '&.custom-button': {
+      'font-size': isLarge ? '1em' : '0.8em',
       'font-weight': '400',
-      'line-height': '2.8em',
+      'line-height': '2.6em',
       padding: '0 1em',
-      height: '2.8em',
+      height: '2.6em',
       'border-radius': `${props.theme.borderRadius.sm}`,
```
Comment (vitalii-retel) [inline src/components/common/custom-button.js:16]: Should it also be set according to the size?

File: src/components/event/dashboard/index.js [hunk 2]
```diff
@@ -1,14 +1,59 @@
...
+import styled from 'styled-components';
 import { Container } from '../../common/container';
-import { Headline } from '../../../styles/typography';
+import { Welcome } from './welcome';
+import { Share } from './share';
+import { CurrentPlan } from './current-plan';
+import { Registrations } from './registrations';
+import { Modules } from './modules';
+
+const DashboardWrapper = styled(Container)`
```
Comment (vitalii-retel) [inline src/components/event/dashboard/index.js:10]: In my opinion we are creating a complex hierarchy of html elements and styles. I would try to create each component independent as much as possible. I can imagine some of generic styles/components. But let's use descriptive name for such components. For example here, Container - what is it? :) And what if something is changed in Container - then this component will be affected too.

Comment (vitalii-retel) [reply src/components/event/dashboard/index.js:10]: Let's just try to keep everything under control. Please take a look at better names for generic component which are reusable. Then local name could be standardised too. Like Wrapper etc. I don't think each component needs each own local names like Component1Wrapper, Cmp2Wrapper etc. Let's just use the same local names if that has the same meaning (like an example with Wrapper).

File: src/styles/typography.js [hunk 3]
```diff
@@ -16,14 +15,21 @@ export const Title = styled.h2`
   line-height: 1.17;
   color: ${(props) => props.theme.palette.text.primary};
   font-weight: 500;
-  letter-spacing: -0.01em;
 `;
 
-export const Subheading = styled.h3`
+export const Subheading1 = styled.h3`
```
Comment (vitalii-retel) [inline src/styles/typography.js:20]: 1 and 2 not a nice name, can we use something more speakable - for example, small, big etc..

**Review by vitalii-retel — APPROVED** [review]

File: src/components/common/custom-button.js (hunk 1 above)
Comment (oleynichenko) [reply src/components/common/custom-button.js:16]: no, it is depend on font-size which was changed before

File: src/styles/typography.js (hunk 3 above)
Comment (oleynichenko) [reply src/styles/typography.js:20]: ok

File: src/components/event/dashboard/index.js (hunk 2 above)
Comment (oleynichenko) [reply src/components/event/dashboard/index.js:10]: >I would try to create each component independent as much as possible. 
I try to keep this approach. styled(Container)` - it is the only exception with other paddings in design. 

>For example here, Container - what is it? 
Will rename it to PageContainer

>  And what if something is changed in Container - then this component will be affected too.
yes, that is fine. Container is used only as page wrapper

> Like Wrapper etc. I don't think each component needs each own local names like Component1Wrapper, Cmp2Wrapper etc. Let's just use the same local names if that has the same meaning (like an example with Wrapper).

All which are named with prefix - means that it is defined in the same file. All others  - imported. I would keep it if it is not very important. I just very used to "верстать покомпонентно"

File: src/styles/typography.js (hunk 3 above)
Comment (oleynichenko) [reply src/styles/typography.js:20]: Material Design spec like these :-). I will fix

File: src/components/common/custom-button.js (hunk 1 above)
Comment (oleynichenko) [reply src/components/common/custom-button.js:16]: no? it is depend on font-size. It is changed above

**Pull request merged**